app/controllers/purchase-orders.js

```diff
@@ -1,34 +1,36 @@
-import Ember from 'ember';
-import computed from 'ember-computed-decorators';
+import Ember from "ember";
+import computed from "ember-computed-decorators";
 
-const { filterBy } = Ember.computed;
 const tomorrow = moment().add(1, "days").format("YYYY-MM-DD");
 
 export default Ember.Controller.extend({
   deliveryDate: tomorrow,
 
-  @computed('orders.@each.{deliveryDate}', 'deliveryDate')
+  @computed("orders.@each.{deliveryDate}", "deliveryDate")
```

Comment:
We need to add `isPurchaseOrder` as a key.

Basically anything that is used in the computed prop should be added up in the computed.